.env.example

```
@@ -1,154 +1,188 @@
+# ── DynamoDB Evidence Metadata ──
+# NOTE: Backend uses DDB_* prefix, AI Worker uses DYNAMODB_* prefix
+# Keep both in sync!
+DDB_EVIDENCE_TABLE=leh_evidence_dev
+DDB_CASE_SUMMARY_TABLE=leh_case_summary_dev
+DYNAMODB_TABLE=leh_evidence_dev
+DYNAMODB_TABLE_CASE_SUMMARY=leh_case_summary_dev
+
+# ── Qdrant Vector Store ──
+QDRANT_HOST=localhost
+QDRANT_PORT=6333
+QDRANT_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The QDRANT_API_KEY key should go before the QDRANT_HOST key

(UnorderedKey)